core/src/main/java/org/apache/carbondata/core/statusmanager/FileFormatProperties.java

```diff
@@ -17,10 +17,19 @@
 
 package org.apache.carbondata.core.statusmanager;
 
+import java.util.HashSet;
+import java.util.Set;
+
 /**
  * Provides the constant name for the file format properties
  */
 public class FileFormatProperties {
+  public static Set<String> SUPPORTED_EXTERNAL_FORMAT = new HashSet<String>() {
```

Comment:
better to make it private and provide function to validate whether user input format is supported

Reply:
OK

Reply:
OK